app/search/views.py

```diff
@@ -1,11 +1,20 @@
-# HOW TO DENY ACCESS TO ES PORT FOR ANYTHING BUT APP?
-
-from flask import request, jsonify, render_template
-
+from flask import (
+    request,
+    jsonify,
+    render_template,
+)
+from flask_login import current_user, login_user
 from app import es
 from app.search import search
-from app.search.constants import INDEX
-from app.lib.utils import eval_request_bool
+from app.search.constants import (
+    INDEX,
+    DEFAULT_HITS_SIZE,
+)
+from app.lib.utils import (
+    eval_request_bool,
+    InvalidUserException,
+)
+# TODO: HOW TO DENY ACCESS TO ES PORT FOR ANYTHING BUT APP?
```

Comment:
In Production, the ES cluster will sit behind an Nginx Reverse Proxy that will only allow connections from specified IP addresses. We can also use https://www.elastic.co/products/x-pack/security